libs/ui-lib/lib/common/components/hosts/HostStatus.tsx

```diff
@@ -214,8 +214,12 @@ const WithHostStatusPopover: React.FC<WithHostStatusPopoverProps> = (props) => (
   </Popover>
 );
 
-const getHostStatusIcon = (icon: React.ReactNode, progress: HostProgressInfo | undefined) => {
-  if (progress?.stageTimedOut !== undefined) {
+const getHostStatusIcon = (
+  icon: React.ReactNode,
+  progress: HostProgressInfo | undefined,
+  status: HostStatusProps['status'],
+) => {
+  if (progress?.stageTimedOut !== undefined && status.title !== 'Error') {
```

Comment:
Will there be a mention of what caused the error?

Since with this change the message _Waiting for control plane has been active more than the expected completion time_ would not be shown from the UI

Reply:
Yes, the error is shown in other place. This is only to don't show the warning with timeout when an error happens.